kubernetes/apps/prow.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Should we add license headers to Argo manifests?

Reply:
We don't have to. +1 for leaving it out

Reply:
Okay, +1 from my side as well to leave it out.